### PR TITLE
Improve Area support in default templates

### PR DESCRIPTION
**Pull request by phenning**
@javiercn @mlorbetske @sayedihashimi @joeloff

### CODE REVIEW

**Comment by phenning** [comment]
Fixes #591

**Comment by joeloff** [comment]
:shipit:

**Comment by phenning** [comment]
@rynowak, added the publish options change to this now as well

File: src/BaseTemplates/StarterWeb/project.json [hunk 1]
```diff
@@ -71,6 +71,7 @@ $endif$
     "include": [
       "wwwroot",
       "Views",
+      "Areas\**\Views",
```
Comment (rynowak) [inline src/BaseTemplates/StarterWeb/project.json:74]: Does this do the right thing? Just making sure you double checked that this is actually the right include

Comment (phenning) [inline src/BaseTemplates/StarterWeb/project.json:74]: Yup this works, verified in a sample app

Comment (phenning) [inline src/BaseTemplates/StarterWeb/project.json:74]: Actually the include should be Areas/**/Views / instead of \

**Pull request merged**